.github/workflows/ci.yml

```diff
@@ -180,3 +345,21 @@ jobs:
           source tee_launcher/venv/bin/activate
           cd tee_launcher 
           PYTHONPATH=. pytest -vsx
+
+  run-tests:
+    name: "Run tests"
+    runs-on: warp-ubuntu-2204-x64-8x
+    timeout-minutes: 60
+    needs:
+      - docker-build
+      - clippy-mpc-node
+      - clippy-mpc-devnet
+      - clippy-mpc-node-tee
+      - clippy-mpc-contract
+      - mpc-tee-unittests
+      - mpc-unittests
+      - mpc-contract-unittests
+      - mpc-pytests
+      - tee-launcher-tests
+    steps:
+      - run: echo "All tests passed."
```

Comment:
Added this to satisfy the branch protection rule.

Alternatively, we could also change the branch protection rule (c.f. [this thread](https://github.com/orgs/community/discussions/26698#discussioncomment-3252954))

<img width="1068" height="729" alt="image" src="https://github.com/user-attachments/assets/4f50fb0f-f167-4bdb-9355-e62c7ebcf82e" />